README.md

```diff
@@ -1,46 +1,111 @@
+## 📝 Table of Contents:
 
-## 🚀 Quick start
+1. [Getting Started](https://github.com/Scout-NU/vital-site#start)
+   - [Cloning](https://github.com/Scout-NU/vital-site#clone)
+   - [Environment Variables](https://github.com/Scout-NU/vital-site#env)
+   - [npm Modules](https://github.com/Scout-NU/vital-site#npm)
+2. [Deployment](https://github.com/Scout-NU/vital-site#deploy)
+   - [How to Manage Netlify Environment Variables](https://github.com/Scout-NU/vital-site#netlifyenv)
+   - [How to Deploy Project on Netlify](https://github.com/Scout-NU/vital-site#deploy2)
+3. [Contentful](https://github.com/Scout-NU/vital-site#contentful)
+4. [Gatsby](https://github.com/Scout-NU/vital-site#gatsby)
+5. [Config Files](https://github.com/Scout-NU/vital-site#config)
+6. [Scout Developers](https://github.com/Scout-NU/vital-site#devs)
 
-1.  **Create a Gatsby site.**
+## 🚦 <a name="start">Getting Started </a>
 
-    Use the Gatsby CLI to create a new site, specifying the default starter.
+1.  **<a name="clone">Clone the repository</a>**
 
-    ```shell
-    # create a new Gatsby site using the default starter
-    gatsby new my-default-starter https://github.com/gatsbyjs/gatsby-starter-default
-    ```
+```shell
+git clone git@github.com:Scout-NU/vital-site.git
+```
 
-1.  **Start developing.**
+2.  **Change directories**
 
-    Navigate into your new site’s directory and start it up.
+```shell
+cd vital-site
+```
 
-    ```shell
-    cd my-default-starter/
-    gatsby develop
-    ```
+3.  **<a name="env">Set up environment variables</a>**
 
-1.  **Open the source code and start editing!**
+In the project directory is a file called `env_example`. This holds all of the necessary environment variables to create this project, including the Contentful access token.
 
-    Your site is now running at `http://localhost:8000`!
+Copy the file twice, name the new files .env.development and .env.production, and add the Contentful API keys to the variables. You can find these API keys in your Contentful space under Settings -> API keys -> vital-site. You only need to use Space ID and Content Delivery API - access token. Do **not** commit these to the Github repo
 
-    _Note: You'll also see a second link: _`http://localhost:8000/___graphql`_. This is a tool you can use to experiment with querying your data. Learn more about using this tool in the [Gatsby tutorial](https://www.gatsbyjs.com/tutorial/part-five/#introducing-graphiql)._
+```shell
+cp env_example .env.development
+```
 
-    Open the `my-default-starter` directory in your code editor of choice and edit `src/pages/index.js`. Save your changes and the browser will update in real time!
+```shell
+# In .env
+CONTENTFUL_SPACE_ID=''
+CONTENTFUL_ACCESS_TOKEN=''
+```
 
-## 🧐 What's inside?
+4. **<a name="npm">Install the npm modules</a>**
+
+```shell
+# Inside the project directory
+npm install
+```
+
+Run server/development environment:
+
+```shell
+npm run dev
```

Comment:
This doesn't appear to be a command, I think it should be `develop` (or you can change it to `dev` in `package.json`)